packaging/arch/PKGBUILD

```diff
@@ -3,28 +3,34 @@
 # Contributor: Zygmunt Krynicki <me at zygoon dot pl>
 
 pkgbase=snapd
-pkgname=(snapd snap-confine)
-pkgver=2.26.1
+pkgname=snapd
+pkgver=2.26.14.r1445.g3df03f98b
 pkgrel=1
 arch=('i686' 'x86_64')
 url="https://github.com/snapcore/snapd"
 license=('GPL3')
-makedepends=('git' 'go' 'go-tools' 'bzr')
-checkdepends=('python' 'squashfs-tools')
-
-# snap-confine
-makedepends+=('libcap' 'python-docutils' 'systemd' 'xfsprogs')
-checkdepends+=('indent' 'shellcheck')
+makedepends=('git' 'go' 'go-tools' 'libseccomp' 'libcap' 'python-docutils' 'systemd' 'xfsprogs' 'libseccomp')
+checkdepends=('python' 'squashfs-tools' 'indent' 'shellcheck')
 
 options=('!strip' 'emptydirs')
 install=snapd.install
-source=("git+https://github.com/snapcore/$pkgname.git#tag=$pkgver"
-        'snapd.sh')
+source=("git+https://github.com/snapcore/$pkgname.git"
+        'snapd.sh'
+        '0001-release-remove-default-from-VERSION_ID.patch'
+        '0001-cmd-snap-confine-don-t-share-etc-nsswitch-from-host.patch')
 md5sums=('SKIP'
```

Comment:
It means that `makepkg` won't try to checksum the tarball coming out of git master (since it keeps moving)